environment/action.yaml

```diff
@@ -32,22 +35,25 @@ runs:
             environment_type=feature
             environment_name=$(echo ${{ inputs.environment }} | sed 's/feature\///' | tr '[:upper:]' '[:lower:]')
             github_branch=feature/$(echo ${{ inputs.environment }} | sed 's/feature\///' | tr '[:lower:]' '[:upper:]')
+            short_environment_name=$(echo ${{ inputs.environment }} | sed 's/feature\///' | sed -e 's/-//')
```

Comment:
Could possible reduce repetition by using a global statement (since we have `$environment_type` available.

Something like:
`echo "short_environment_name=$(echo ${{ inputs.environment }} | sed -E "s/${environment_type}\///" | tr -d '-')" >> $GITHUB_OUTPUT` 

Since the SED includes the `/` it won't match on `develop` et-al where the name is _just_ the environment